backstory/src/demo/java/com/google/sps/servlets/GPT2Servlet.java

```diff
@@ -38,6 +37,8 @@
   
   public static final int DEFAULT_MAX_STORY_LENGTH = 200;
   public static final Double DEFAULT_TEMPERATURE = 0.7;
+  private StoryManagerURLProvider URLProvider;
```

Comment:
I think this field needs to be static and synchronized, since multiple servlet instances may be created. Maybe this would be a solution:
https://wiki.sei.cmu.edu/confluence/display/java/LCK05-J.+Synchronize+access+to+static+fields+that+can+be+modified+by+untrusted+code

Reply:
Done